report_thesis/src/sections/background.tex

```diff
@@ -5,7 +5,18 @@ \subsubsection{Standard Scaler}
 
 \subsubsection{Max Absolute Scaler}
 
-\subsubsection{MinMax Scaler}
+\subsubsection{Min-Max Normalization}
+Min-max normalization rescales the range of features to $[0, 1]$ or $[a, b]$. 
```

Comment:
you use $a$ and $b$ but don't reuse them in the formula below. i'd say that would be better than $new max$ and $new min$.